src/Workspaces/Core/Portable/Remote/RemoteHostSessionHelpers.cs

```diff
@@ -273,12 +270,19 @@ public void Shutdown()
             }
 
             var connection = await client.TryCreateConnectionAsync(_serviceName, _callbackTarget, cancellationToken).ConfigureAwait(false);
-            if (connection == null)
+            try
```

Comment:
📝 Ideally you would just place the previous line in a `using` statement, but this will not work due to CS1657. See #26313.

/cc @jcouv @OmarTawfik